lib/src/migrators/module/references.dart

```diff
@@ -46,16 +45,14 @@ class References {
   ///
   /// Each value in this map must be a [VariableDeclaration] or an [Argument].
   final BidirectionalMap<MemberDeclaration<VariableDeclaration>,
-          MemberDeclaration<SassNode /*VariableDeclaration|Argument*/ >>
```

Comment:
These shadow union types are still as informative now as they were before, right?

Reply:
Now that the actual getters used required are covered by `SassDeclaration`, I don't think there's any case where we actually have to do an instance check, so I thought it might be more readable to remove the union type comment and just specify what it is in the doc comment.

Would you prefer to keep the comments?

Reply:
Either way is fine with me.